lib/src/models/network.dart

```diff
@@ -525,15 +525,15 @@ class OmniXepNetwork implements BasedUtxoNetwork {
 
   /// Retrieves the Human-Readable Part (HRP) for Pay-to-Witness-Public-Key-Hash (P2WPKH) addresses.
   @override
-  String get p2wpkhHrp => throw const DartBitcoinPluginException(
-      "Omni XEP network does not support P2WPKH/P2WSH");
+  String get p2wpkhHrp => conf.params.p2wpkhHrp!;
 
   /// Checks if the current network is the mainnet.
   @override
   bool get isMainnet => true;
 
   @override
   final List<BitcoinAddressType> supportedAddress = const [
+    SegwitAddresType.p2wpkh,
```

Comment:
Missing an "s" in address.

Reply:
This is the existing Enum

Reply:
https://github.com/mrtnetwork/bitcoin_base/blob/main/lib/src/bitcoin/address/core.dart

Reply:
> https://github.com/mrtnetwork/bitcoin_base/blob/main/lib/src/bitcoin/address/core.dart

hi, you must add config in ```blockchain_utils``` for ```BIP-44``` and ```BIP-89``` confing, or if we have any testnet i can testing  and add all of them to the package.

Reply:
@mrtnetwork, done!

Reply:
@mrtnetwork I've added BIP-44, BIP-84, and the test net.